src/rhsmlib/dbus/util.py

```diff
@@ -77,8 +77,6 @@ def dbus_handle_exceptions(func, *args, **kwargs):
         pattern = "^HTTP error \x28.*\x29: "
         err_msg = re.sub(pattern, "", str(err))
         # Modify severity of some exception here
-        if "Ignoring request to auto-attach. It is disabled for org" in err_msg:
-            severity = "warning"
```

Comment:
this actually used to be an error message from candlepin itself, so potentially something that matched here; the whole stringification of the exception thrown during a D-Bus call is wonky (the severity, the regexp matching, ...), so I'd say to leave this as-is for now